### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by SugarWTF** [cinit/QAuxiliary#409] [Q&A] [answered]
使用qauxiliary一定需要root吗

**Comment by CJYKK** [comment, marked as answer]
> 使用qauxiliary一定需要root吗

不一定，用weishu大佬的太极就行

**Reply by SugarWTF** [reply]
成功了，谢谢

**Reply by ShirakamiAme** [reply]
太极的稳定性和兼容性仍然诡异，推荐lspatch打包便携模式，顺带提一句微信别做，签名对不上在某些app的诸如充值，购物的场景下微信会无法正常唤起

**Reply by SugarWTF** [reply]
> 太极的稳定性和兼容性仍然诡异，推荐lspatch打包便携模式，顺带提一句微信别做，签名对不上在某些app的诸如充值，购物的场景下微信会无法正常唤起

我在太极下用qua和哔哩漫游，没有修改微信的东西。没有碰到什么东西用不了的问题，就是qq偶尔会失去响应，重启进程就好了
